plugins/pa11y/readme.md

```diff
@@ -0,0 +1,30 @@
+# @dotcom-tool-kit/pa11y
+
+A plugin to run [Pa11y](https://github.com/pa11y/pa11y) accessibility tests. This plugin uses [Pa11y CI](https://github.com/pa11y/pa11y-ci) to run the tests.
+
```

Comment:
sorry just one last thing following the changes I added in https://github.com/Financial-Times/dotcom-tool-kit/pull/279/files: 

**suggestion**: would it be possible to mention here that the `TEST_URL` environment variable gets populated with the value of the heroku review app, so users can use `process.env.TEST_URL` in their .pa11yci config file?

Reply:
Done 👍 I have made a new section of `## To note` (which definitely needs improving!). Let me know what you think and whether you think it should be somewhere else.